start.sh

```diff
@@ -42,6 +42,20 @@ prep() {
     ${COMPOSE} pull
 }
 
+# Gets the IP of the local machine.
+getIp() {
+    HOSTNAME_VAL="$(hostname -i)"
```

Comment:
@dekobon just to be clear, my point here is that `hostname -i` should be unreachable code unless you're in an environment where a) there's no Triton config and b) there's no Docker Machine. Which means you're using Docker on Linux, *or* you're using some crazy unsupported thing like xhyve.